app/Http/Controllers/CustomerController.php

```diff
@@ -13,7 +14,111 @@ class CustomerController extends Controller
+   
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {   
+        
+        $imageName = time().'.'.$request->image->extension();  
+        $request->image->storeAs('customers', $imageName);
+        
+        $customers = new Customers();
+        $customers->name =  $request->name;
+        $customers->tin_number =  $request->tin_number;
+        $customers->address =  $request->address;
+        $customers->city =  $request->city;
+        $customers->country =  $request->country;
+        $customers->mobile_number = $request->mobile_number;
+        $customers->telephone_one = $request->telephone_one;
+        $customers->telephone_two = $request->telephone_two;
+        $customers->fax = $request->fax;
+        $customers->website = $request->website;
+        $customers->email = $request->email;
+        $customers->contact_person = $request->contact_person;
+        $customers->label = $request->label;
+        $customers->image =  $imageName;
+        $customers->is_active ='Yes';
+        $customers->save();
+        return back();
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Models\Customers  $customers
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Customers $customers)
+    {
+        $customers = Customers::all();
+
+        // Compute for each inventory value and total value.
+         
+
+        return view('customer.customer.index',compact('customers'));
+    }
+    public function edit($id)
+    {
+        $customers = Customers::find($id);
+        return view('customer.customer.edit', compact('customers'));
+    }
+
+    public function update(Request $request, $id)
+    { 
+        $customers = Customers::find($id);
+
+        $customers->name =  $request->input('name');
+        $customers->tin_number =  $request->input('tin_number');
+        $customers->address =  $request->input('address');
+        $customers->city =  $request->input('city');
+        $customers->country =  $request->input('country');
+        $customers->mobile_number = $request->input('mobile_number');
+        $customers->telephone_one = $request->input('telephone_one');
+        $customers->telephone_two = $request->input('telephone_two');
+        $customers->fax = $request->input('fax');
+        $customers->website = $request->input('website');
+        $customers->email = $request->input('email');
+        $customers->contact_person = $request->input('contact_person');
+        $customers->label = $request->input('label');
+        $customers->is_active ='Yes';
+        $customers->update();
+
+        return redirect()->back();
```

Comment:
Is this supposed to redirect back to `/customer/`? or stay to the `edit` page?

Reply:
It is supposed to stay on the edit page but I will edit it and  redirect it back to the customer

Reply:
I also include the success message
![image](https://user-images.githubusercontent.com/66796707/159835934-7418da0d-4564-45cf-a373-852f67bb84e5.png)

Reply:
Alright, if its supposed to stay in `edit` page, that is fine. Just adding a `success` message will do. 
Redirecting it back to `customer` is also welcome. Anyway, we'll just clarify that scenario to the client. Cheers! 🚀